src/AgGrid.elm

```diff
@@ -1096,6 +1096,16 @@ onSelectionChange nodeDecoder toMsg =
         )
 
 
+{-| Detect model update events, like changing filter values or sorting.
+-}
+onModelDataUpdated : (Result Decode.Error (List String) -> msg) -> Html.Attribute msg
+onModelDataUpdated toMsg =
```

Comment:
The name is quite generic for what it returns.

We should name it more specific to the returned values. Something like `onVisibleRowIdsChanged`